cli/listener/src/main.rs

```diff
@@ -36,24 +45,13 @@ pub struct Opts {
     #[clap(long, default_value = "300")]
     poll_frequency_seconds: u32,
 
-    /// Location of the SQLite DB file.
+    /// Location of the SQLite DB file, defaults to "listener.db".
     #[clap(long, default_value = "listener.db")]
```

Comment:
Nit: I prefer to use `.sqlite3` extension to make it more obvious what type of file it is.